llm/llm.go

```diff
@@ -89,9 +89,10 @@ func New(workDir, model string, adapters []string, opts api.Options) (LLM, error
 		opts.NumGQA = 0
 		opts.RopeFrequencyBase = 0.0
 		opts.RopeFrequencyScale = 0.0
-		return newLlama(model, adapters, chooseRunners(workDir, "gguf"), ggml.NumLayers(), opts)
+		// return newLlama(model, adapters, chooseRunners(workDir, "gguf"), ggml.NumLayers(), opts)
```

Comment:
We can remove all of the `chooseRunners` logic now – this should simplify things quite a bit!

Reply:
Nevermind, this PR already does this!